modules/basegl/src/shadercomponents/volumecomponent.cpp

```diff
@@ -77,9 +77,10 @@ constexpr std::string_view voxel = util::trim(R"(
 )");
 
 constexpr std::string_view gradientFirst = util::trim(R"(
-#if defined(GRADIENTS_ENABLED)
 vec3 {0}GradientPrev = vec3(0);
-vec3 {0}Gradient = useSurfaceNormals ? -texture(surfaceNormal, texCoords).xyz :
+vec3 {0}Gradient = vec3(0);
```

Comment:
Since these variables are potentially used for lighting calculation, even though no gradients are present. Otherwise you end up with a non-compiling shader instead of just diffuse color without illumination.